ganga/GangaCore/__init__.py

```diff
@@ -424,6 +424,7 @@ def getLCGRootPath():
 
 condor_config.addOption('query_global_queues', True,
                  "Query global condor queues, i.e. use '-global' flag")
+condor_config.addOption('MaxBytes', "1000000", 'Max bytes option for peek option for Condor backend ')
```

Comment:
Could you make the docstring a little more helpful
```suggestion
condor_config.addOption('MaxBytes', "1000000", 'Set maximum bytes to display with peek()')
```